src/utils/renderChainPath.test.ts

```diff
@@ -1,38 +1,74 @@
 import { renderChainPath } from './renderChainPath'
 
 describe('renderChainPath', () => {
+  const m = process.version.match(/(\d+)\.(\d+)\.(\d+)/) || ['10.0.0', '10']
+  const [major] = m.slice(1).map(_ => parseInt(_))
+
+  it('should return original', () => {
+    const path = 'foo'
+
+    const result = renderChainPath(path)
+    if (major < 14) {
```

Comment:
I used this to run the tests via Github actions against node 12. So these tests are covered but the snapshots will fail.